pypeit/core/datacube.py

```diff
@@ -1128,13 +1501,30 @@ def coadd_cube(files, opts, spectrograph=None, parset=None, overwrite=False):
     flatpar = parset['calibrations']['flatfield']
     senspar = parset['sensfunc']
 
-    # Get the detector number and string representation
-    det = 1 if parset['rdx']['detnum'] is None else parset['rdx']['detnum']
-    detname = spec.get_det_name(det)
-
     # prep
     numfiles = len(files)
     combine = cubepar['combine']
+    method = cubepar['method'].lower()
+
+    # Determine what method is requested
+    if method == "resample":
+        if shapely is None:
+            msgs.error("To use the 'resample' algorithm to make your datacube, you need to install shapely.")
+        else:
```

Comment:
Don't need the else (`msgs.error` breaks the code flow).

Reply:
Resample is now gone as an option!